pkg/efivarfs/fs_test.go

```diff
@@ -20,32 +20,24 @@ func TestFSGoodFile(t *testing.T) {
 	}
 	i, err := getInodeFlags(f)
 	if err != nil {
-		t.Skipf("Can not getInodeFlags: %v != nil", err)
+		t.Skipf("Cannot getInodeFlags: %v != nil", err)
 	}
 
 	if err := setInodeFlags(f, i); err != nil {
 		t.Fatalf("setInodeFlags: %v != nil", err)
```

Comment:
This might as well be  a skipf, right? This is mainly testing that we can set the inodeflags, and if we can't, well ...  too bad, but it's not really an error in the test?

Reply:
Which one? The commented line is already a Skipf.

Reply:
line 27. There might be some kernel-based reason we can't set inodeflags to their current value, but that's not really our problem, maybe? It just means we need to skip the test, because if we can't set the flags to their current value, there's something else going on. WDYT?

Reply:
anyway, we can wait until this is an issue, this code seems fine.

Reply:
Sorry, had a busy day yesterday. Fwiw, even in my runs when the set-immutable part of the test was being skipped, we could still set the flags to their current value, so I think it will be fine as-is.